Applications/Games/Space Engineers/Steam/script.js

```diff
@@ -0,0 +1,38 @@
+include("engines.wine.quick_script.steam_script");
+include("engines.wine.verbs.dotnet472");
+include("engines.wine.verbs.vcrun2017");
+include("engines.wine.verbs.dxvk");
+include("engines.wine.verbs.faudio");
+include("engines.wine.plugins.override_dll");
+include("utils.functions.filesystem.files");
+
+var installerImplementation = {
+    run: function () {
+        new SteamScript()
+            .name("Space Engineers")
+            .editor("Keen Software House")
+            .author("Zemogiter")
+            .appId(244850)
+            .wineVersion("4.9")
+            .wineDistribution("upstream")
+            .wineArchitecture("amd64")
+            .preInstall(function (wine, wizard) {
+                wine.dotnet472();
+                wine.vcrun2017();
+                wine.DXVK();
+                wine.faudio();
+                wine.overrideDLL()
+                    .set("native, builtin", ["xaudio2_0", "xaudio2_1", "xaudio2_2", "xaudio2_3", "xaudio2_4", "xaudio2_5", "xaudio2_6", "xaudio2_7", "xaudio2_8", "xaudio2_9"])
+                    .do();
+                wizard.message(tr("You have to install libjpeg62 and libjpeg62-dev or else the thumbnails in New Game menu will be dispalyed as magenta rectangles."));
```

Comment:
This should not be necessary thanks to the runtime

Reply:
Do you mean the dll override or the libs?

Reply:
the installation of libjpeg.

Reply:
Weird. I still needed to install those libs to display the thumbnails correctly

Reply:
Weird. I still needed to install those libs to display the thumbnails correctly

Reply:
Could you launch without WINDEBUG=-all and post the wine.log in the two cases ? What I do not understand is that the libs provided by libjpeg62 and libjpeg-dev should be in the runtime.

Reply:
Maybe you should merge master in this branch before ^^.